### PR TITLE
Removes logic required to refresh temporary secrets

### PR DESCRIPTION
**Pull request by iamkirkbater**
This PR aims to satisfy OSD-4088 by removing temporary secrets entirely.

Instead of keeping these secrets on hand they should be generated with the `osdctl` cli.

### CODE REVIEW

**Comment by rogbas** [comment]
We'll merge it as soon as we update the SOP for obtaining the console URL using the osdctl. @clcollins can you please update here with a link for the card?

**Comment by rogbas** [comment]
@iamkirkbater can you please rebase?

**Comment by iamkirkbater** [comment]
Ops SOP PR 873 is on hold and should be merged with this.

File: pkg/controller/account/account_controller.go [hunk 1]
```diff
@@ -369,48 +369,13 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
...
+		_, err = r.BuildIAMUser(reqLogger, awsAssumedRoleClient, currentAcctInstance, iamUserSRE, request.Namespace)
 		if err != nil {
 			r.setStatusFailed(reqLogger, currentAcctInstance, fmt.Sprintf("Failed to build IAM SRE user: %s", iamUserSRE))
 			return reconcile.Result{}, err
 		}
 
-		// Intermittently our secret wont be ready before the next call, lets ensure it exists
-		for i := 0; i < 10; i++ {
-			secret := &corev1.Secret{}
-			err := r.Client.Get(context.TODO(), types.NamespacedName{Name: *SREIAMUserSecret, Namespace: request.Namespace}, secret)
```
Comment (jharrington22) [inline pkg/controller/account/account_controller.go:382]: @iamkirkbater you need to remove `types` from the imports its not used in this file

Comment (jharrington22) [inline pkg/controller/account/account_controller.go:382]: @iamkirkbater we need to remove `types` from the import statements its unused now

**Pull request merged**